investment-time/research.md

```diff
@@ -0,0 +1,9 @@
+# Research
+
+Investment time was proposed, piloted, and recommended outside of the normal
+scope of R&D. As such, artifacts were not drafted in GitHub. They are linked
+here for posterity.
+
+- 🤔 [Proposal](https://docs.google.com/document/d/1HXm9r4afnQGgz3CcddryoS2Al-YqU6I__1zuGtqhr3I/edit?usp=sharing)
+- ✈️ [Pilot](https://docs.google.com/document/d/1lY2qY2X23L8gMiPTRXrUp_f-6AkdTg0saTl3MgzsCWY/edit?usp=sharing)
```

Comment:
This link goes to a document titled `Pilot: Investment Time Proposal`, which I would have expected under the `Proposal` bullet, but I might be misinterpreting here.

Reply:
Fixed the first link. The pilot should go to the document containing the parameters for our pilot. Does this make more sense now?

Reply:
Yeah, this matches my expectations now!